env/staging/replica_count.yaml

```diff
@@ -35,15 +45,15 @@ metadata:
   namespace: notification-canada-ca
 spec:
   minReplicas: 2
-  maxReplicas: 4
+  maxReplicas: 2
```

Comment:
I downgraded this one to replicate production, as staging had more defined than production which I found curious. Tell me if that was intended and I will revert back.

Reply:
We disabled this in production because we didn't test enough HPA for admin/API so we just put 4 API pods and 2 admin pods in production, which is enough by experience